### DISCUSSION TITLE
chat bubble flip

### DISCUSSION THREAD

**Discussion by tkd992006** [saadeghi/daisyui#1688] [Q&A] [answered]
Is it possible to flip the design of the chat bubble component upside down so that pointy part is on top not on bottom? like chat bubble in kakao talk. because it looks better when i use chat bubble with user's profile when chat bubble component is flipped(When the pointy part is on top).
<img width="240" alt="스크린샷 2023-03-14 오후 2 35 08" src="https://user-images.githubusercontent.com/16851261/224906662-575e9c61-6a86-4f12-9fac-78c9d4e45a0a.png">
![스크린샷 2023-03-14 오후 2 35 19](https://user-images.githubusercontent.com/16851261/224906668-bdb6b465-b0dc-4b01-890a-9740a361103e.png)

**Comment by saadeghi** [comment, marked as answer]
You can move the triangle to the top like this:  
https://play.tailwindcss.com/9vaX7xTnt0?file=css
```css
.chat-start .chat-bubble{
  @apply rounded-tl-none rounded-bl-[1rem];
}
.chat-end .chat-bubble{
  @apply rounded-tr-none rounded-br-[1rem];
}
.chat-bubble:before{
  @apply top-0 [transform:rotateX(180deg)];
}
```

Related discussion: https://github.com/saadeghi/daisyui/discussions/1636

Let me know if you have any questions.
